[IMP] Attendances: updating images and formatting

larm-odoo opened the pull request:
Updating to new formatting standards. Images have annotations that need to be removed, one image is extra/unnecessary.

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/12980.png)](https://mergebot.odoo.com/odoo/documentation/pull/12980)

jero-odoo reviewed (approved):
Hey @larm-odoo looks good to me, just a couple of things to note. 
There is another image on this doc that has not been updated but still has the red markups on it (we are trying to move away from marked up images) 
![image](https://github.com/user-attachments/assets/cdc116da-048a-4cc7-815a-0123a91f3cb9)
Also you will want to assign this doc to yourself!

larm-odoo commented:
Hi @jero-odoo - thanks for reminding me to assign myself- I hadn't done that for ANY of my PR's =D.
I also changed this to a draft when I realized I needed to recreate that image, but needed several hours os it looked like I worked a few hours, etc. So I changed it to a draft, but had already tagged you- so I apologize for that, that was all my fault! It was only that one image you called out that was holding this back from not being a draft. Thank you for bearing with me while my feet are getting re-wet as they say!

larm-odoo commented:
Hi @samueljlieber - this is ready for a tech review. Thank you!

Felicious commented on content/applications/hr/attendances/check_in_check_out.rst:
```diff
@@ -2,19 +2,15 @@
-.. image:: check_in_check_out/top-menu.png
-   :align: center
-   :alt: Top right main menu with check in button highlighted.
+of what application the user is in, a :guilabel:`🔴 (red circle)` or :guilabel:`🟢 (green circle)` is
```
I know we phased out the emojis, but looking at the HTML for the icon, the circle icon has no color ):

Are there workarounds for this @samueljlieber, or do you think we should just fly with the emojis?

![image](https://github.com/user-attachments/assets/4283243a-4d26-4e62-8d97-ef100632b91b)
 
![image](https://github.com/user-attachments/assets/fe2751cb-c2a3-4f36-8aa7-df80379a3a1b)

samueljlieber replied:
@Felicious @larm-odoo 
I recommend we describe the color like so, formatting the :icon: as usual:
```suggestion
of what application the user is in, a :icon:`fa-circle` :guilabel:`(red circle)` or
:icon:`fa-circle` :guilabel:`(green circle)` is
```
LMK your thoughts!

---

Coloring icons is possible, and we discussed it when first implementing icons here: https://github.com/odoo/documentation/pull/8469#pullrequestreview-2024398296

We decided not to use colors in icons or text in favor of writing out the UI color changes.

For your reference, this is one way we could color icons in documentation. Notice in your screenshot the `text-success` class that follows the icon's class. This is one of the [Bootstrap color classes](https://getbootstrap.com/docs/4.0/utilities/colors/#color). The following RST will not work without first modifying the :icon: role's definition.

```rst
of what application the user is in, a :icon:`fa-circle text-danger` :guilabel:`(red circle)` or
:icon:`fa-circle text-success` :guilabel:`(green circle)` is 
```
<img width="896" alt="Screenshot 2025-04-22 at 2 31 09 PM" src="https://github.com/user-attachments/assets/3370c9e2-94a1-4fa9-af3c-8bb9d1c09bd8" />

A larger discussion is needed around the standards/guidelines for icon colors and how we want to implement them in documentation :)

larm-odoo replied:
These are both good and valid.  I personally prefer the color only because the UI is colored, and seeing a black circle then describing it as 'red' or 'green' is not my favorite- but it IS consistent with how we do other icons. When I wrote this doc, I know this was discussed, and we had originally landed on the colored circle icons that were in here, and I left them in and did not update them since I  thought it was an exception, since we hashed it out when this was created. 

Since we have the colored dots available, I do like the second option @samueljlieber presented, but I will absolutely just use the black dot and describe it, if that's what we want to do, @Felicious. Not sure who to listen to here, so I will await a reply to this comment =D

larm-odoo replied:
I JUST re-read this, and see that the colored dots are not visible unless we make some modifications. Since that's never a good ide-a I put in the black dots and the text descriptor. Sorry about the confusion!

Felicious commented on content/applications/hr/attendances/check_in_check_out.rst:
```diff
@@ -2,19 +2,15 @@
-   :align: center
-   :alt: Top right main menu with check in button highlighted.
+of what application the user is in, a :guilabel:`🔴 (red circle)` or :guilabel:`🟢 (green circle)` is
+visible. Click on the colored circle to reveal the attendance widget, enabling the user to check-in
+and/or out.
```
"check-in" is a noun, so let's remove the hyphen to make it a verb (:

I might be overthinking this, but i don't think you can be checked in AND out at the same time, which is why I removed the "and"
```suggestion
visible. Click on the colored circle to reveal the attendance widget, enabling the user to check in
or out.
```

Felicious commented on content/applications/hr/attendances/check_in_check_out.rst:
```diff
@@ -73,16 +68,15 @@ attendance widget updates again, displaying the :guilabel:`Total today` field wi
-When the user checks out from the database, the *Attendances* app logs the location details for the
-user. This information is **only** logged if the user allows their computer to access this
+When the user checks out from the database, the **Attendances** app logs the location details for
+the user. This information is **only** logged if the user allows their computer to access this
 information.
```
suggestion for brevity:

> This information is **only** logged if the user allows their computer to access this
information.

to This information is logged **only** if the user grants permission.

Felicious commented on content/applications/hr/attendances/check_in_check_out.rst:
```diff
@@ -73,16 +68,15 @@ attendance widget updates again, displaying the :guilabel:`Total today` field wi
-   check in and out without any time elapsing (a value of 00:00). Each time an employee logs in and
-   out, the information is stored and appears on the main *Attendances* dashboard, including check
-   ins and check outs with no time value.
+   The number of times a user can check-in and check-out is unlimited. Users can check-in and out
```
let's change "check-in" to "check in" in this paragraph, when using it as a verb (:

```suggestion
   The number of times a user can check in and checkout is unlimited. Users can check in and out

Felicious commented on content/applications/hr/attendances/check_in_check_out.rst:
```diff
@@ -73,16 +68,15 @@ attendance widget updates again, displaying the :guilabel:`Total today` field wi
-   ins and check outs with no time value.
+   The number of times a user can check-in and check-out is unlimited. Users can check-in and out
+   without any time elapsing (a value of 00:00). Each time an employee logs in and out, the
+   information is stored and appears on the main **Attendances** dashboard, including check ins and
```
gotta add a hyphen this time since it's a noun! 
```suggestion
   information is stored and appears on the main **Attendances** dashboard, including check-ins and
```

Felicious commented on content/applications/hr/attendances/check_in_check_out.rst:
```diff
@@ -73,16 +68,15 @@ attendance widget updates again, displaying the :guilabel:`Total today` field wi
+   The number of times a user can check-in and check-out is unlimited. Users can check-in and out
+   without any time elapsing (a value of 00:00). Each time an employee logs in and out, the
+   information is stored and appears on the main **Attendances** dashboard, including check ins and
+   check outs with no time value.
```
```suggestion
   check-outs with no time value.
```

Felicious reviewed (approved):
LGTM @larm-odoo !

larm-odoo commented:
Hi @samueljlieber - I believe this is now yours! I re-read your comment regarding the dots (I did leave those in on purpose, because I thought it was an exception since it was hashed out a while ago). I changed them so they are both just black dots with descriptions.

larm-odoo commented:
@robodoo r+
